opennlp-tools/lang/es/abb_ES.xml

```diff
@@ -0,0 +1,236 @@
+-->
+
+<dictionary case_sensitive="false">
+  <entry>
+    <token>a.C.</token>
+  </entry>
+  <entry>
+    <token>a. de C.</token>
+  </entry>
+  <entry>
+    <token>a.J.C.</token>
+  </entry>
+  <entry>
+    <token>a. de J.C.</token>
+  </entry>
+  <entry>
+    <token>a. m.</token>
+  </entry>
+  <entry>
+    <token>apdo.</token>
+  </entry>
+  <entry>
+    <token>apdo.</token>
+  </entry>
+  <entry>
+    <token>aprox.</token>
+  </entry>
+  <entry>
+    <token>Av.</token>
+  </entry>
+  <entry>
+    <token>Avda.</token>
+  </entry>
+  <entry>
+    <token>Bs. As.</token>
+  </entry>
+  <entry>
+    <token>c.c.</token>
+  </entry>
+  <entry>
+    <token>cap.</token>
+  </entry>
+  <entry>
+    <token>D.</token>
+  </entry>
+  <entry>
+    <token>Da.</token>
+  </entry>
+  <entry>
+    <token>Dña.</token>
+  </entry>
+  <entry>
+    <token>d.C.</token>
+  </entry>
+  <entry>
+    <token>d. de C.</token>
+  </entry>
+  <entry>
+    <token>d.J.C.</token>
+  </entry>
+  <entry>
+    <token>d. de J.C</token>
+  </entry>
+  <entry>
+    <token>dna.</token>
+  </entry>
+  <entry>
+    <token>EE. UU.</token>
+  </entry>
+  <entry>
+    <token>etc.</token>
+  </entry>
+  <entry>
+    <token>f.c.</token>
+  </entry>
+  <entry>
+    <token>F.C.</token>
+  </entry>
+  <entry>
+    <token>FF. AA.</token>
+  </entry>
+  <entry>
+    <token>Dr.</token>
+  </entry>
+  <entry>
+    <token>Dra.</token>
+  </entry>
+  <entry>
+    <token>Gob.</token>
+  </entry>
+  <entry>
+    <token>Lic.</token>
+  </entry>
+  <entry>
+    <token>Ing.</token>
+  </entry>
+  <entry>
+    <token>Pdte.</token>
+  </entry>
+  <entry>
+    <token>Pdta.</token>
+  </entry>
+  <entry>
+    <token>pág.</token>
+  </entry>
+  <entry>
+    <token>no.</token>
+  </entry>
+  <entry>
+    <token>núm.</token>
+  </entry>
+  <entry>
+    <token>p.ej.</token>
+  </entry>
+  <entry>
+    <token>p. m.</token>
+  </entry>
+  <entry>
+    <token>Prof.</token>
+  </entry>
+  <entry>
+    <token>Profa.</token>
+  </entry>
+  <entry>
+    <token>q.e.p.d.</token>
+  </entry>
+  <entry>
+    <token>S.A.</token>
+  </entry>
+  <entry>
+    <token>S.L.</token>
+  </entry>
+  <entry>
+    <token>Sr.</token>
+  </entry>
+  <entry>
+    <token>Sra.</token>
+  </entry>
+  <entry>
+    <token>Srta.</token>
+  </entry>
+  <entry>
+    <token>Ud.</token>
+  </entry>
+  <entry>
+    <token>Vd.</token>
+  </entry>
+  <entry>
+    <token>Uds.</token>
+  </entry>
+  <entry>
+    <token>Vds.</token>
+  </entry>
+  <entry>
+    <token>vol.</token>
+  </entry>
+  <entry>
+    <token>v.</token>
+  </entry>
+  <entry>
+    <token>lu.</token>
+  </entry>
+  <entry>
+    <token>ma.</token>
+  </entry>
+  <entry>
+    <token>mi.</token>
+  </entry>
+  <entry>
+    <token>ju.</token>
+  </entry>
+  <entry>
+    <token>vi.</token>
+  </entry>
+  <entry>
+    <token>sá.</token>
+  </entry>
+  <entry>
+    <token>do.</token>
+  </entry>
+  <entry>
+    <token>en.</token>
+  </entry>
+  <entry>
+    <token>feb.</token>
```

Comment:
febr. in Spain, I believe. Will have to check with some native speakers.

https://www.fundeu.es/escribireninternet/abreviaturas-de-los-meses-y-los-dias-de-la-semana/